Formula/mopidy-beets.rb

```diff
@@ -6,15 +6,15 @@ class MopidyBeets < Formula
   head "https://github.com/mopidy/mopidy-beets.git"
   revision 1
 
-  depends_on "python@3.11"
+  depends_on "python@3.12"
```

Comment:
Optimally, do the upgrade of Python in one commit and the upgrade of Mopidy in another.

Reply:
Got it 👍